### PR TITLE
Docker scripts for Ubuntu Focal/Galactic and Jammy/Humble

### PR DESCRIPTION
**Pull request by spham-amzn**
Dockerscript files to build and run simulations and robot applications for the loft scene robotic simulation demo.

### CODE REVIEW

File: Docker/LaunchRViz.bash.ubuntu-galactic [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```
Comment (adamdbrw) [inline Docker/LaunchRViz.bash.ubuntu-galactic:1]: I suggest changing the file name:
- This actually launches ROS 2 navigation stack
- RViz2 is just an addition to the stack

Comment (adamdbrw) [reply Docker/LaunchRViz.bash.ubuntu-galactic:1]: LaunchNavStack?

**Review by adamdbrw — COMMENTED** [review]
Looks very nice, one minor naming suggestion

File: Docker/Dockerfile.navstack.ubuntu-galactic [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                                                         ros-${ROS_DISTRO}-nav2-bringup \
+                                                         ros-${ROS_DISTRO}-pointcloud-to-laserscan \
+                                                         ros-${ROS_DISTRO}-gazebo-msgs \
+                                                         ros-${ROS_DISTRO}-ackermann-msgs \
+                                                         ros-${ROS_DISTRO}-control-toolbox'
+
+# Install the O3DE-specific required packages
+RUN apt-get install -y --no-install-recommends libglu1-mesa-dev \
+                                               mesa-common-dev \
+                                               libnvidia-gl-470
```
Comment (lumberyard-employee-dm) [inline Docker/Dockerfile.navstack.ubuntu-galactic:43]: Isn't libnvidia-gl-470 kind of old?
It looks like it goes up to 520 now https://packages.ubuntu.com/search?suite=jammy&section=all&arch=any&keywords=libnvidia-gl&searchon=names

File: Docker/Dockerfile.simulation.ubuntu-galactic [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    cd o3de-demo-project && \
+    git lfs install && \
+    git lfs pull && \
+    $WORKSPACE/o3de/scripts/o3de.sh register -pp $WORKSPACE/o3de-demo-project/  && \
+    cd $WORKSPACE/o3de-demo-project && \
+    /bin/bash -c 'source /opt/ros/galactic/setup.bash && \
+                  cmake -B build/linux -S . -G "Ninja Multi-Config" -DLY_STRIP_DEBUG_SYMBOLS=TRUE -DLY_DISABLE_TEST_MODULES=ON && \
+                  cmake --build build/linux --config profile --target AssetProcessorBatch ROS2-Gem-Demo.GameLauncher ROS2-Gem-Demo.Assets && \
+                  cd $WORKSPACE && \
+                  ./cleanup.bash'
```
Comment (lumberyard-employee-dm) [inline Docker/Dockerfile.simulation.ubuntu-galactic:105]: You can change the directory for git command using the -`C` option (https://git-scm.com/docs/git#Documentation/git.txt--Cltpathgt)

File: Docker/Dockerfile.navstack.ubuntu-galactic (hunk 2 above)
Comment (spham-amzn) [reply Docker/Dockerfile.navstack.ubuntu-galactic:43]: There seems to be a problem with libnvidia-gl-520 (vulkan is not detected  in the container). 470 is known to work

**Review by lumberyard-employee-dm — APPROVED** [review]

**Pull request merged**